doc/developer/development-cycle.html

```diff
@@ -25,18 +25,21 @@ <h2>Prepare your working copy</h2>
 
 <li>
 To begin work on crux, you will need a copy of the code base.
-<a href="https://git-scm.com/docs/git-clone">Clone</a> a clean clean 
+<a href="https://git-scm.com/docs/git-clone">Clone</a> a clean  
 copy from the
 <a href="https://github.com/crux-toolkit/crux-toolkit">GitHub repository</a>
 or update your existing
-working copy.  Some changes can be made directly in the trunk, but if
-you are working on something experimental or sufficiently complicated,
-you can create a <a href="https://git-scm.com/docs/git-branch">branch</a>, do
-your work there, and later merge your changes into the trunk.
-
+working copy.  Unless you are a senior developer on the team it is highly recommended to create 
```

Comment:
I would change "Unless you are a senior developer on the team" to "For most changes".